content/en/docs/tutorials/cluster-management/install-use-dra.md

```diff
@@ -0,0 +1,540 @@
+learn more about ResourceClaims. 
+
+### Create the ResourceClaim
+
+The Pod manifest itself will include a reference to its relevant ResourceClaim
+object, which you will create now. Whatever the claim, the `deviceClassName` is
+a required field, narrowing down the scope of the request to a specific device
+class. The request itself can include a {{< glossary_tooltip term_id="cel" >}}
+expression that references attributes that may be advertised by the driver
+managing that device class. 
+
+In this example, you will create a request for any GPU advertising over 10Gi
+memory capacity. The attribute exposing capacity from the example driver takes
+the form `device.capacity['gpu.example.com'].memory`. Note also that the name of
+the claim is set to `some-gpu`.
+
+{{% code_sample language="yaml" file="dra/driver-install/example/resourceclaim.yaml" %}}
+
+```shell
+kubectl apply --server-side -f http://k8s.io/examples/dra/driver-install/example/resourceclaim.yaml
+```
+
+### Create the Pod that references that ResourceClaim
+
+Below is the Pod manifest referencing the ResourceClaim you just made,
+`some-gpu`, in the `spec.resourceClaims.resourceClaimName` field. The local name
+for that claim, `gpu`, is then used in the
+`spec.containers.resources.claims.name` field to allocate the claim to the Pod's
+underlying container.
+
+{{% code_sample language="yaml" file="dra/driver-install/example/pod.yaml" %}}
+
+```shell
+kubectl apply --server-side -f http://k8s.io/examples/dra/driver-install/example/pod.yaml
+```
+
+### Explore the DRA state
+
+The cluster now tries to schedule that Pod to a node where Kubernetes can
+satisfy the ResourceClaim. In our situation, the DRA driver is deployed on all
+nodes, and is advertising mock GPUs on all nodes, all of which have enough
+capacity advertised to satisfy the Pod's claim, so this Pod may be scheduled to
+any node and any of the mock GPUs on that node may be allocated.
+
+The mock GPU driver injects environment variables in each container it is
+allocated to in order to indicate which GPUs _would_ have been injected into
+them by a real resource driver and how they would have been configured, so you
+can check those environment variables to see how the Pods have been handled by
+the system.
+
+1.  Confirm the pod has deployed:
+
+    ```shell
+    kubectl get pod pod0 -n dra-tutorial
+    ```
+
+    The output is similar to this:
+    ```
+    NAME   READY   STATUS    RESTARTS   AGE
+    pod0   1/1     Running   0          9s
+    ```
+
+1.  Observe the pod logs which report the name of the mock GPU allocated:
+
+    ```shell
+    kubectl logs pod0 -c ctr0 -n dra-tutorial | grep -E "GPU_DEVICE_[0-9]+=" | grep -v "RESOURCE_CLAIM"
+    ```
+
+    The output is similar to this:
+    ```
+    declare -x GPU_DEVICE_4="gpu-4"
+    ```
+
+1.  Observe the ResourceClaim object:
+
+    You can observe the ResourceClaim more closely, first only to see its state
+    is allocated and reserved.
```

Comment:
Move this after the output block. like "In this output, the `STATE` column shows that the ResourceClaim is allocated and reserved."

Reply:
Fixed in a follow up PR